recipe_scrapers/_schemaorg.py

```diff
@@ -114,6 +114,15 @@ def ingredients(self):
 
     def nutrients(self):
         nutrients = self.data.get("nutrition", {})
+
+        # Some recipes contain null or numbers which breaks normalize_string()
+        # We'll ignore null and convert numbers to a string, like Schema validator does
+        for key, val in nutrients.items():
+            if val is None:
+                del nutrients[key]
```

Comment:
Ah; hmm.. there is one small problem here.  This deletion results in a `RuntimeError` because the code is modifying the `nutrients` dictionary while iterating over it.